<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::GeneralHelpPreferencesEditor</class>
 <widget class="QWidget" name="tt3::gui::GeneralHelpPreferencesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="cacheHelpCollectionCheckBox">
     <property name="text">
      <string>Cache help collection</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="viewHelpInLabel">
     <property name="text">
      <string>View help in:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="systemBrowserRadioButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>System browser</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">helpAgentButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="helpVieverRadioButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Help viewer</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">helpAgentButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>220</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cacheHelpCollectionCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>tt3::gui::GeneralHelpPreferencesEditor</receiver>
   <slot>_cacheHelpCollectionCheckBoxStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>systemBrowserRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::GeneralHelpPreferencesEditor</receiver>
   <slot>_systemBrowserRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpVieverRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::GeneralHelpPreferencesEditor</receiver>
   <slot>_helpViewerRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_cacheHelpCollectionCheckBoxStateChanged(int)</slot>
  <slot>_systemBrowserRadioButtonClicked()</slot>
  <slot>_helpViewerRadioButtonClicked()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="helpAgentButtonGroup"/>
 </buttongroups>
</ui>
